ci/test.py

```diff
@@ -1091,6 +1104,11 @@ def test_bb_env_extrawhite_parse(self):
         )
         self.assertEqual(s, 'TEST_BB_EXTRA="foo"')
 
+    @skipIfOS("ubuntu", "14.04")
+    @skipIfOS("ubuntu", "16.04")
+    def test_wine(self):
+        self.assertPyrexContainerCommand("wine --version")
```

Comment:
This test (rolled in here from the former derived class) doesn't seem to really be doing anythig other than proving that dpkg didn't fail during Docker image building.

I wonder whether having to define the conditional test annotations and truck those around, is more trouble than the test is worth.

Reply:
I'd prefer to keep it since it validates something didn't get removed or missed when adding a new image (which has happened before).